source/product-recipes/sample-product-recipes.rst

```diff
@@ -0,0 +1,19 @@
+Problem Statement
```

Comment:
This document appears to be missing a main title. Also this one should be added to the table of contents.

Also, is the purpose of this page to provide example recipes?

Reply:
Yes.